### DISCUSSION TITLE
Need help with offer canceling

### DISCUSSION THREAD

**Discussion by lukar777** [Chia-Network/chia-blockchain#15843] [Support]
Long ago I created a nft collection and put them on dexie, they are marked as Legacy offers. https://twitter.com/dexie_space/status/1683126860815777792?s=20
I can't see them no more in the gui as my offers. How can i cancel them?

**Comment by BrandtH22** [comment]
Hey @lukar777 , as a heads up the majority of our support currently is taking place in our new discord server:
https://discord.gg/chia

As for cancelling offers that you cannot see int he GUI, one guaranteed way to cancel an offer is to send the associated coins to yourself, this "spends" the coins in the offer thereby making it invalid.

ex. If you created an offer to sell an NFT you would need to send that NFT to yourself to cancel the offer.
ex. On the flip side if you created an offer to buy an NFT you would need to send your xch coins to yourself.

You can find a valid receive address for yourself at the top of the GUI or on the receive tab under the xch token section.
